bot/exts/evergreen/connect_four.py

```diff
@@ -0,0 +1,466 @@
+import asyncio
+import random
+import typing
+from functools import partial
+
+import discord
+import emojis
+from discord.ext import commands
+
+NUMBERS = [
```

Comment:
I think all of these emojis should be in `bot.constants` as they're not necessarily specific to this cog.